content/en/docs/Responses/structuredLyrics.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "structuredLyrics"
+linkTitle: "structuredLyrics [OS]"
+opensubsonic:
+  - Addition
+description: >
+  Structured lyrics
+---
+
+{{< tabpane persistLang=false >}}
+{{< tab header="**Example**:" disabled=true />}}
+{{< tab header="OpenSubsonic JSON" lang="json">}}
+{
+  "displayArtist": "Muse",
+  "displayTitle": "Hysteria",
+  "lang": "eng",
+  "offset": -100,
+  "synced": true,
+  "line": [
+    {
+      "start": 0,
+      "value": "It's bugging me"
+    },
+    {
+      "start": 2000,
+      "value": "Grating me"
+    },
+    {
+      "start": 3001,
+      "value": "And twisting me around..."
+    }
+  ]
+}
+{{< /tab >}}
+{{< tab header="OpenSubsonic XML" lang="xml">}}
+<structuredLyrics displayArtist="Muse" displayTitle="Hysteria" lang="en" offset="-100" synced="true">
+  <line start="0">It's bugging me</line>
+  <line start="2000">Grating me</line>
+  <line start="3001">And twisting me around...</line>
+</structuredLyrics>
+{{< /tab >}}
+{{< tab header="Subsonic"  >}}
+Does not exist.
+{{< /tab >}}
+{{< /tabpane >}}
+
+| Field           | Type                       | Req.    | OpenS.  | Details                                                                                                                                                                |
+| --------------- | -------------------------- | ------- | ------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `lang`          | `string`                   | **Yes** | **Yes** | The lyrics language                                                                                                                                                    |
+| `synced`        | `boolean`                  | **Yes** | **Yes** | True if the lyrics are synced, false otherwise                                                                                                                         |
+| `line`          | Array of [`line`](../line) | **Yes** | **Yes** | The actual lyrics. Ordered by start time (synced) or appearance order (unsynced)                                                                                       |
+| `displayArtist` | `string`                   | No      | **Yes** | The artist name to display. This could be the localized name, or any other value                                                                                       |
```

Comment:
You already have the song ID since you need it to request ;) Those values can be present in the lyrics themselves so are exposed

Reply:
I don't really see the need for the displayX values at all then - I don't understand what you mean by:
> Those values can be present in the lyrics themselves

... but I'm an API minimalist so...

Reply:
As you can see they are optional ... And as said a lyric file can have headers that contains information, this is the information that is exposed here. If the information is available then it can be exposed.

Reply:
This was probably noted, but per [Wikioedia](https://en.wikipedia.org/wiki/LRC_(file_format) you can have fields like
```lrc
[ti:Lyrics (song) title]
[ar:Lyrics artist]
```
which could be the artist/song title, localized. Since those are not the same as the artist ID (and are only for display purposes), it would be nice to expose them (if they exist)